src/commands/Owner/Avatar.js

```diff
@@ -27,11 +27,10 @@ class Avatar extends Command {
    * @param {Message} message Message with a command to handle, reply to,
    *                          or perform an action based on parameters.
    */
-  run(message) {
+  async run(message) {
     const url = message.strippedContent.match(this.regex)[1];
-    this.bot.client.user.setAvatar(url)
-    .then(() => this.messageManager.reply(message, 'New avatar set!', true, true))
-    .catch(this.logger.error);
+    await this.bot.client.user.setAvatar(url);
+    this.messageManager.reply(message, 'New avatar set!', true, true);
```

Comment:
could you add the success returns for when it succeeds when the message is sent?